content/accessibility/text-resize-and-respacing.mdx

```diff
@@ -0,0 +1,75 @@
+---
+title: Text resize and respacing
+---
+
+## Overview
+
+Users should be able to resize text and use custom text spacing without text being clipped, truncated, or obscured. In addition, interactive controls should not
+break.
+
+## Why?
+
+People with low-vision may rely on the browser's zoom functionality or update the font-size of a page to read content. Additionally, people with low-vision
+and people with dyslexia may increase text spacing with custom CSS to improve readability. 
+
+When sites are built without adequate responsivity considerations (e.g. setting fixed width and height, using non-relative font sizing), sites may become completely unreadable 
+and unusable for some users.
+
+## How to test 
+
+### Text resizing
+
+Most modern browsers support full-page zoom, while some (e.g. FireFox) also support text-only zoom. While WCAG does not specify how text should be resized, it is best practice for your 
+page to be readable with full-page zoom and text-only zoom. Aside from using the browser zoom, some users may also resize text through browser font-size setting.
+
+Test your page by using browser zoom with magniciation of up to 200%. Observe that the text content does not overflow, overlap, or become truncated, and that interactive controls still work.
+
+Learn more about resize testing techniques at [TheA11yProject: Resize text](https://www.a11yproject.com/posts/resize-text/). 
+
+#### Text spacing
+
+Modify text spacing according to the [WCAG text spacing testing guidelines](https://www.w3.org/WAI/WCAG21/Understanding/text-spacing.html). You can do this
+by changing the CSS using the browser inspector or by using a tool like [Steve Faulkner: text spacing bookmarklet](https://cdpn.io/stevef/debug/YLMqbo).
+
+Observe that text is contained within the bounds of it's container without overlapping or being cutoff.
+
+## Guidelines
+
+### For designers
+
```

Comment:
Hey @langermank and @vdepizzol! I heard that you're doing work related to Primer font-size conversion to REM system. Is there any guidance you think we should offer here to designers and developers meanwhile?